docs/CONTRIBUTING.md

```diff
@@ -0,0 +1,17 @@
+# Contribution Guidelines
```

Comment:
Allthough it's not mentioned anywhere in https://github.com/src-d/guide/blob/master/engineering/documentation.md#contributingmd

I belive it's customary to put this in the root dir of the project.

Reply:
Apparently in docs it will also be [picked up by github](https://help.github.com/articles/setting-guidelines-for-repository-contributors/).

But I don't care one way or another, I'll move it back to the root folder.